api/application.proto

```diff
@@ -216,6 +216,11 @@ message ListApplicationCollaboratorsRequest {
   uint32 limit = 2 [(validate.rules).uint32.lte = 1000];
   // Page number for pagination. 0 is interpreted as 1.
   uint32 page = 3;
+  // Order the results by this field path (must be present in the field mask).
+  // Default ordering is by ID. Prepend with a minus (-) to reverse the order.
+  string order = 4 [
+    (validate.rules).string = { in: ["", "rights", "_id", "-rights", "-_id"] }
```

Comment:
I understand that this is `_id` since it can be either user ID or organization ID, but I still still prefer just plain old `id`.

Reply:
I originally went with `id` but from testing locally the console provides the `_id` to the URL `https://tti.staging1.cloud.thethings.industries/api/v3/applications/adrian-test-lab/collaborators?page=1&limit=20&query=&order=_id`

I can make it `id` but it has to change on the console as well.